Makefile

```diff
@@ -25,40 +25,29 @@ export DEFAULT_ROUTING ?= basic
 export KUBECONFIG ?= ${HOME}/.kube/config
 export DEVWORKSPACE_API_VERSION ?= 1f335562c475972132851c68227dd36558317bb3
 
+# Set GO_PACKAGE_PATH to what is found in go.mod
+GO_PACKAGE_PATH ?= $(shell head -n1 go.mod | cut -d " " -f 2)
```

Comment:
can we move it closer to targets where it's used `compile_...` or even more to internal target, like `_eval_build_info`

Reply:
sorry i didn't realize there was a comment that said #internal params :woman_facepalming:  would it be okay if i moved it where the rest of the internal params are maybe?

Reply:
I know this one is outdated, but I think it makes sense to keep all the top-level vars in one place; we've gotten confused by putting this stuff in a rule before. Variable declarations like this are processed before any rules.

Reply:
Okay, will keep it at the top. I've added it with the other top-level vars